validations/accounts.test.ts

```diff
@@ -35,33 +35,65 @@ describe('accounts', () => {
     const getAccountsResult = await fiatConnectClient.getFiatAccounts()
     expect(getAccountsResult.isOk).to.be.true
     expect(!!getAccountsResult.unwrap().BankAccount?.length).to.be.false
-    await checkObjectAgainstModel(getAccountsResult.unwrap(), 'GetFiatAccountsResponse')
+    await checkObjectAgainstModel(
+      getAccountsResult.unwrap(),
+      'GetFiatAccountsResponse',
+    )
   })
-  it('able to post and get', async () => {
+  it('able to post, get, and delete', async () => {
     const wallet = ethers.Wallet.createRandom()
     const fiatConnectClient = new FiatConnectClient(
       {
         baseUrl: config.baseUrl,
         network: Network.Alfajores,
         accountAddress: wallet.address,
-        apiKey: config.clientApiKey
+        apiKey: config.clientApiKey,
       },
       (message: string) => wallet.signMessage(message),
     )
     const loginResult = await fiatConnectClient.login()
     expect(loginResult.isOk).to.be.ok
 
+    // Add an account and verify response
     const addAccountResult = await fiatConnectClient.addFiatAccount(
       mockAccountData,
     )
     expect(addAccountResult.isOk).to.be.true
-    await checkObjectAgainstModel(addAccountResult.unwrap(), 'FiatAccountInfoResponse')
+    await checkObjectAgainstModel(
+      addAccountResult.unwrap(),
+      'FiatAccountInfoResponse',
+    )
 
+    // Get account and verify that added account is there
     const getAccountsResult = await fiatConnectClient.getFiatAccounts()
     expect(getAccountsResult.isOk).to.be.true
-    await checkObjectAgainstModel(getAccountsResult.unwrap(), 'GetFiatAccountsResponse')
-
+    await checkObjectAgainstModel(
+      getAccountsResult.unwrap(),
+      'GetFiatAccountsResponse',
+    )
     expect(getAccountsResult.unwrap().BankAccount?.length).to.be.gt(0)
+
+    // Delete account
+    const fiatAccountId = addAccountResult.unwrap().fiatAccountId
+    const deleteAccountResult = await fiatConnectClient.deleteFiatAccount({
+      fiatAccountId,
+    })
+    expect(deleteAccountResult.isOk).to.be.true
+
+    // Should fail when attempting to delete account a second time
+    const deleteAccountResultDuplicate =
+      await fiatConnectClient.deleteFiatAccount({ fiatAccountId })
+    expect(deleteAccountResultDuplicate.isOk).to.be.false
+
+    if (deleteAccountResultDuplicate.isErr) {
+      expect(deleteAccountResultDuplicate.error.fiatConnectError).to.be.equal(
+        FiatConnectError.ResourceNotFound,
+      )
+    }
+
+    // Getting accounts should now yield empty response
+    const getAccountsResultDuplicate = await fiatConnectClient.getFiatAccounts()
+    expect(getAccountsResultDuplicate.isOk).to.be.true
+    expect(getAccountsResultDuplicate.unwrap()).to.eql({})
```

Comment:
It would probably be safer order of operations to do this /get check before doing the second /delete check

Reply:
good callouts all around! updated tests to move the order of the checks.